rotkehlchen/globaldb/handler.py

```diff
@@ -1305,13 +1305,26 @@ def get_all_manual_latest_prices(
             return read_cursor.fetchall()
 
     @staticmethod
-    def delete_manual_latest_price(asset: Asset) -> None:
+    def delete_manual_latest_price(asset: Asset) -> Asset:
         """
-        Deletes manual current price from globaldb.
+        Deletes manual current price from globaldb and returns the asset that was being used as
+        reference for the price
         May raise:
         - InputError if asset was not found in the price_history table
         """
         with GlobalDBHandler().conn.write_ctx() as write_cursor:
+            # check the asset of reference that will be deleted
+            write_cursor.execute(
```

Comment:
I have a question. This only invalidates for one possible pair of the `from_asset` or am I wrong?
e.g If you have EUR -> USD, EUR -> CHY... you're only invalidating for EUR -> USD per `Asset(to_asset[0])` 

Or are you expecting it always to be only one pair?

Reply:
yes. At any moment in the database there is at most only one row with the type `manual latest price` and `from_asset` the one provided.

Reply:
Yeah seen. Thanks